README.rst

```diff
@@ -36,6 +70,10 @@ Looking for more help?
 - Chat with us on `Slack`_
 - Get `paid support`_
 
+.. _Download the jar: https://dl.bintray.com/crate/crate/io/crate/crate-jmx-exporter/0.1.0/crate-jmx-exporter-0.1.0.jar
```

Comment:
Should we link to the specific version or to the parent folder? (I'm not sure)

Reply:
imho we should link to a JAR file so the user can just download it. otherwise the user does not completely know which jar (javadocs, source) to download

Reply:
yep, ok!